security/transformer.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 The Service Manager Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package security
+
+// EncryptionTransformer implements CredentialsTransformer by encrypting and decrypting the credentials
+type EncryptionTransformer struct {
```

Comment:
Looks like it could be removed. It's a simple implementation of CredentialsTransformer interface which would be useful if other transformations are introduced (e.g. hashing passwords).